crates/core/src/operations/load.rs

```diff
@@ -81,11 +86,23 @@ impl std::future::IntoFuture for LoadBuilder {
                 .transpose()?;
 
             let ctx = SessionContext::new();
+            
+            // Parquet options, if any, are set in the SessionContext
+            let mut state = ctx.state();
+            if let Some(parquet_config) = this.parquet_config {
+                let pc: ParquetConfig = ParquetConfig::from_str(parquet_config.as_str()).unwrap();
+                let mut table_config = TableOptions::new();
+                table_config.set_config_format(ConfigFileType::PARQUET);
+                table_config.parquet = pc.try_into().unwrap();
+                let tom = state.table_options_mut();
+                *tom = table_config;
+            }
+            
```

Comment:
See comment in crates/core/src/delta_datafusion/mod.rs for how we access these crypto settings. We may be able to do this differently for how we put the crypto settings in the state.